src/core/xlio.h

```diff
@@ -127,7 +127,7 @@ int xlio_pipe(int __filedes[2]);
 
 int xlio_open(__const char *__file, int __oflag, ...);
 
-int xlio_creat(const char *__pathname, mode_t __mode);
+int xlio_creat(const char *__pathname, __mode_t __mode);
```

Comment:
can you explain this change?

Reply:
`xlio_creat` in static build is invoked instead of `creat`. Internally the exported symbol calls the original function.
The `__mode` variable is passed to the original syscall. The original function signature looks as follows:
`int creat(const char *pathname, mode_t mode);`

**The fix seems correct to me**

Reply:
please fix according to the online discussion, thanks.

Reply:
removed